man/tpm2_commit.1.md

```diff
@@ -0,0 +1,82 @@
+% tpm2_commit(1) tpm2-tools | General Commands Manual
+
+# NAME
+
+**tpm2_commit**(1) - Performs the first part of an ECC anonymous signing
+operation.
+
+# SYNOPSIS
+
+**tpm2_commit** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_commit**(1) - Performs the first part of an ECC anonymous signing
+operation. The TPM will perform the point multiplications on the provided points
+and return intermediate signing values. The signHandle parameter shall refer to
+an ECC key and the signing scheme must be anonymous (TPM_RC_SCHEME).
```

Comment:
i don't see a signHandle and I am also not getting what you mean by "signign scheme anonymous". Are you referring to the EC Key Objects scheme parameters being NULL? Below it looks like it needs an ECDAA scheme, is that what you meant?

Reply:
Yes it should be a ECDAA. Also the sign handle is the key and its auth specified with -p and -c. I will correct the text.